app/assets/stylesheets/helpers/_footer.scss

```diff
@@ -18,31 +18,24 @@
     text-decoration: none;
   }
 
-  .footer-categories {
-    @extend %contain-floats;
-    @extend %grid-row;
+  .footer-container {
+    @include media(tablet) {
+      padding-top: 60px;
```

Comment:
Is it worth using `govuk-spacing()`?

Reply:
This is interesting - I wanted to do that, but while `static` includes the components gem, it doesn't pull in any sass (to avoid duplication where apps do that and use static) so I can't use any `govuk-frontend` mixins or variables in static. Fun times.

Reply:
Ouch! Fun times indeed.